types/reply.d.ts

```diff
@@ -14,6 +14,8 @@ export interface ReplyGenericInterface {
 
 type HttpCodesReplyType = Partial<Record<HttpKeys, unknown>>
 
+type HttpHeader = 'accept' | 'accept-charset' | 'accept-encoding' | 'accept-language' | 'accept-ranges' | 'access-control-allow-credentials' | 'access-control-allow-headers' | 'access-control-allow-methods' | 'access-control-allow-origin' | 'access-control-expose-headers' | 'access-control-max-age' | 'access-control-request-headers' | 'access-control-request-method' | 'age' | 'allow' | 'authorization' | 'cache-control' | 'cdn-cache-control' | 'connection' | 'content-disposition' | 'content-encoding' | 'content-language' | 'content-length' | 'content-location' | 'content-range' | 'content-security-policy' | 'content-security-policy-report-only' | 'cookie' | 'dnt' | 'date' | 'etag' | 'expect' | 'expires' | 'forwarded' | 'from' | 'host' | 'if-match' | 'if-modified-since' | 'if-none-match' | 'if-range' | 'if-unmodified-since' | 'last-modified' | 'link' | 'location' | 'max-forwards' | 'origin' | 'prgama' | 'proxy-authenticate' | 'proxy-authorization' | 'public-key-pins' | 'public-key-pins-report-only' | 'range' | 'referer' | 'referrer-policy' | 'refresh' | 'retry-after' | 'sec-websocket-accept' | 'sec-websocket-extensions' | 'sec-websocket-key' | 'sec-websocket-protocol' | 'sec-websocket-version' | 'server' | 'set-cookie' | 'strict-transport-security' | 'te' | 'trailer' | 'transfer-encoding' | 'user-agent' | 'upgrade' | 'upgrade-insecure-requests' | 'vary' | 'via' | 'warning' | 'www-authenticate' | 'x-content-type-options' | 'x-dns-prefetch-control' | 'x-frame-options' | 'x-xss-protection' | string;
```

Comment:
Do we have some automation for this list?

> public-key-pins

First time in my life I read it, but:

> in 2017 browsers deprecated and removed HPKP support

https://en.wikipedia.org/wiki/HTTP_Public_Key_Pinning

I'm afraid we will get PRs to add whatever HTTP header in the world